aws-api-payload.json

```diff
@@ -0,0 +1,209 @@
+                    "spark.sql.warehouse.dir": "s3://$ADG_S3_PUBLISH_BUCKET/data/external",
+                    "spark.sql.catalogImplementation": "hive"
+                },
+                "Classification": "spark-defaults"
+            },
+            {
+                "Properties": {
+                    "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory"
+                },
+                "Classification": "spark-hive-site"
+            },
+            {
+                "Properties": {
+                    "hive.metastore.schema.verification": "false",
+                    "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory",
+                    "hive.metastore.warehouse.dir": "s3://$ADG_S3_PUBLISH_BUCKET/hive/external"
+                },
+                "Classification": "hive-site"
+            },
+            {
+                "Properties": {
+                    "hbase.server.keyvalue.maxsize": "750000000",
+                    "hbase.bulkload.retries.retryOnIOException": "true",
+                    "hbase.client.keyvalue.maxsize": "750000000",
+                    "hbase.client.write.buffer": "8388608",
+                    "hbase.bucketcache.bucket.sizes": "5120,9216,17408,33792,41984,50176,58368,66560,99328,132096,197632,263168,394240,525312,656384,787456,918528,1049600,1180672,1311744,1442816,1704960,1967104,2229248",
+                    "hbase.rootdir": "s3://$ADG_S3_INGEST_BUCKET/business-data/single-topic-per-table-hbase",
+                    "hbase.regionserver.handler.count": "4"
+                },
+                "Classification": "hbase-site"
+            },
+            {
+                "Properties": {
+                    "hbase.emr.storageMode": "s3",
+                    "hbase.emr.readreplica.enabled": "true"
+                },
+                "Classification": "hbase"
+            },
+            {
+                "Properties": {
+                    "fs.s3.cse.materialsDescription.enabled": "true"
+                },
+                "Classification": "emrfs-site"
+            }
+        ],
+        "ReleaseLabel": "emr-5.24.1",
+        "NormalizedInstanceHours": 16,
```

Comment:
We won't need to pass this in either; it's calculated by the EMR service for billing purposes. These 'internal' attributes will become more obvious, I think, when we start trying to marshal this into the `runJobFlow()` API call.